packages/w3up-client/package.json

```diff
@@ -114,13 +117,16 @@
     "access": "public"
   },
   "files": [
-    "dist"
+    "dist",
+    "*.min.js"
   ],
   "scripts": {
     "attw": "attw --pack .",
     "lint": "tsc --build && eslint '**/*.{js,ts}' && prettier --check '**/*.{js,ts,yml,json}' --ignore-path ../../.gitignore",
     "lint:fix": "tsc --build && eslint '**/*.{js,ts}' --fix && prettier --write '**/*.{js,ts,yml,json}' --ignore-path ../../.gitignore",
-    "build": "tsc --build",
+    "build": "npm run build:tsc && npm run build:bundle:browser",
+    "build:tsc": "tsc --build",
+    "build:bundle:browser": "esbuild src/index.js --bundle --minify --target=chrome130 --format=iife --global-name=w3up --outfile=browser.min.js",
```

Comment:
Should we call the global `StorachaClient`?

Reply:
Good catch. I just updated that.